docs/2.10.0/docs/canton/usermanual/troubleshooting_guide.rst

```diff
@@ -325,6 +325,79 @@ Common patterns from the canton log:
 
           The log line contains the name of the missing claim, but not the actual claims. When using tokens based on user names (`Audience Based Tokens <https://docs.daml.com/app-dev/authorization.html#audience-based-tokens>`_ and `Scope Based Tokens <https://docs.daml.com/app-dev/authorization.html#scope-based-tokens>`_), consult the user management service to see whether you need to grant more rights to the user. When using tokens based on party names (`Custom Claims Access Tokens <https://docs.daml.com/app-dev/authorization.html#custom-daml-claims-access-tokens>`_), debug the token in `JWT.IO <https://jwt.io/#debugger-io>`_.
 
+Disconnections
+--------------
+
+Vast majority of disconnections occurring in the real life scenarios are caused by intermittent network failures. Less frequently, however,
+disconnections happen because the network proxies, load balancers, or other elements of the network infrastructure terminate the client-server
+connection because they think that there is no activity going on in the communication channel. Such disconnections are predictable and
+can be avoided by carefully configuring the components' keepalive parameters.
+
+For the GRPC protocol stack, there are two keepalive mechanisms to consider. There is the low level TCP keep-alive feature and then there are the HTTP/2 pings.
+Canton endpoints utilize the latter. It is worthwhile getting familiar with the basics of this mechanism. The article describing
+`how to use HTTP/2 PING-based keepalives in <https://grpc.io/docs/guides/keepalive/>`_ is a good place to start.
+
+The keepalive parameters have to be set on all components involved. In this eample we will concentrate on configuring the Ledger API, similar
+approach can be used for other Canton APIs. The way to do it is described in the dedicated :ref:`keepalive article <keepalive-configuration>`.
+Next, the corresponding parameters need to be altered on the Ledger API client. Please consult the user manual specific to your application.
+The parameters on both ends should be set consistently. Special attention must be given to the fact that the keepalive time setting on the client
```

Comment:
> Special attention must be given to the fact that the keepalive

Maybe let's mark this text (and possibly around it) so it's more visible (for example `:warning` or bold).

Reply:
The text has been reworded and as a consequence the phrase starting with "Special attention" has been made less prominent